plugin/evm/vm.go

```diff
@@ -243,14 +209,20 @@ func (vm *VM) Initialize(
 		return err
 	}
 
-	// Set log level
-	logLevel, err := log.LvlFromString(vm.config.LogLevel)
+	vm.ctx = ctx
+
+	// Create logger
+	alias, err := vm.ctx.BCLookup.PrimaryAlias(vm.ctx.ChainID)
+	if err != nil {
+		alias = vm.ctx.ChainID.String()
```

Comment:
can we log an error here (or is this expected to occur as a fallback and in that case can we clarify with a comment)?

Reply:
done